docs/userguides/config.md

```diff
@@ -37,6 +37,55 @@ plugin:
 
 This helps keep your secrets out of Ape!
 
+If a configuration is left unset (i.e., not included in the
```

Comment:
thanks for the docs! don't be alarmed if i push some nitty commits here before merge